cloud/deploys/bin/azure/key-vault.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+
+#######################################
+# Create resource group
```

Comment:
suggestion (blocking): this function doesn't contain any key vault-specific logic, recommend rewording the doc comments